<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>478</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>332</width>
    <height>478</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>50</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}

background-color : white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>20</y>
      <width>268</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">Qlabel 
{
qproperty-alignment: 'AlignVCenter | AlignRight';
border: 1px solid gray;
}

backgroung-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>131</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusminus">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
    border-radius: 10px;
    text-decoration: none;
    color: black;
    background: #CFCFCF;
	font-weight: bold;
}
QPushButton:hover
{
	background: #ABABAB;
    position: relative;
    top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
